packages/treasury/bundles/install-on-chain.js

```diff
@@ -74,42 +88,65 @@ export async function installOnChain({
     }),
   );
 
+  const electorateTerms = { committeeName: 'TreasuryBoard', committeeSize: 5 };
```

Comment:
Until we work out how anybody should get access to the voter invitations, let's make it 1; that's sort of more obviously not a real committee.

```suggestion
  const electorateTerms = { committeeName: 'TreasuryBoard', committeeSize: 1 };
```

Reply:
I've drafted a no-action electorate. I think that's more clear that it does nothing.